runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -81,6 +83,7 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 			return;
 		}
 
+		int outerZoomLimit = Math.max(-zoomConfig.outerLimit(), OUTER_ZOOM_LIMIT);
```

Comment:
Maybe this should be `128 - zoomConfig.outerLimit()` so setting the config to 1 doesn't change the limit massively.